tests/WE2E/run_WE2E_tests.py

```diff
@@ -160,7 +159,13 @@ def run_we2e_tests(homedir, args) -> None:
         # test-specific options, then write resulting complete config.yaml
         starttime = datetime.now()
         starttime_string = starttime.strftime("%Y%m%d%H%M%S")
-        test_name = os.path.basename(test).split('.')[1]
+        test_fn = os.path.basename(test)
```

Comment:
Is there a specific reason you wanted to stray from the existing convention for test filenames?

Reply:
The original naming convention allowed dots only right after the initial "config" and immediately before the "yaml" extension.  For long test names, I wanted to have another separator character besides the underscore to make the test name more readable.  For example, the new test config file
```
config.get_obs_hpss.do_vx_det.multicyc.cycintvl_24hr_inits_00z_fcstlen_03hr.ncep-hrrr.yaml
```
was originally named 
```
config.get_obs_hpss_do_vx_det_multicyc_cycintvl_24hr_inits_00z_fcstlen_03hr_ncep-hrrr.yaml
```
which I thought was quite confusing to read/understand.  So I wanted to have another separator character for the major portions of the test name.  The dot seemed the most appropriate, but you can see I've also used a dash.  I'm open to suggestions for another naming convention.

Reply:
I think the problem is trying to cram a whole bunch of info about a test in the filename, which is always going to lead to a mess. I acknowledge you didn't start this problem (our existing test names aren't great either) but I think a better solution than making things even more unwieldy is to only include essential disambiguating information in the filename. We have the "description" field that can give more details, and we can grep config options to find out tests that have particular options enabled/disabled. 

I pushed one possible change [here](https://github.com/gsketefian/ufs-srweather-app/pull/3/commits/95795ae465afa3c07225c9e8b178ec70dc5ca980); it includes the minimum info needed to tell the tests apart. I really think we could just go as far as `vx_test_1`, `vx_test_2`, etc. and have all the test info in the description, but that would have to be a larger conversation. 

Since it doesn't show up well in the diffs apparently, these are the suggested renames:

 - config.get_obs_hpss.do_vx_det.singlecyc.init_00z_fcstlen_36hr.winter_wx.SRW.yaml --> config.vx-det_long-fcst_winter-wx_SRW-staged.yaml
 - config.get_obs_hpss.do_vx_det.multicyc.cycintvl_07hr_inits_vary_fcstlen_09hr.ncep-hrrr.yaml --> config.vx-det_multicyc_fcst-overlap_ncep-hrrr.yaml
 - config.get_obs_hpss.do_vx_det.multicyc.cycintvl_24hr_inits_00z_fcstlen_03hr.ncep-hrrr.yaml --> config.vx-det_multicyc_first-obs-00z_ncep-hrrr.yaml
 - config.get_obs_hpss.do_vx_det.multicyc.cycintvl_24hr_inits_21z_fcstlen_03hr.ncep-hrrr.yaml --> config.vx-det_multicyc_last-obs-00z_ncep-hrrr.yaml
 - config.get_obs_hpss.do_vx_det.multicyc.cycintvl_96hr_inits_12z_fcstlen_48hr.nssl-mpas.yaml --> config.vx-det_multicyc_long-fcst-no-overlap_nssl-mpas.yaml
 - config.get_obs_hpss.do_vx_det.multicyc.cycintvl_24hr_inits_12z_fcstlen_48hr.nssl-mpas.yaml --> config.vx-det_multicyc_long-fcst-overlap_nssl-mpas.yaml
 - config.get_obs_hpss.do_vx_det.multicyc.cycintvl_24hr_inits_12z_fcstlen_03hr.nssl-mpas.yaml --> config.vx-det_multicyc_no-00z-obs_nssl-mpas.yaml
 - config.get_obs_hpss.do_vx_det.multicyc.cycintvl_11hr_inits_vary_fcstlen_03hr.ncep-hrrr.yaml --> config.vx-det_multicyc_no-fcst-overlap_ncep-hrrr.yaml